heudiconv/convert.py

```diff
@@ -34,7 +34,7 @@
 lgr = logging.getLogger(__name__)
 
 
-def conversion_info(subject, outdir, info, filegroup, ses):
+def conversion_info(subject, outdir, info, filegroup, ses,echo=None):
```

Comment:
not that we are all "in clear" but it is better to stay `pep8` friendly in the new code -- please add a space before `echo=` here and also where you call it below, but remove around ` = ` when passing as a keyword argument.

I would recommend configuring your IDE/editor to autohighlight pep8 gotchas -- pycharm does it quite beautifully

Reply:
duly noted